inst/test_backends/test_MsBackend/test_spectra_variables.R

```diff
@@ -0,0 +1,286 @@
+#' This test suite runs tests for accessing (and eventually setting) spectra
+#' variables on a provided `MsBackend` instance and validates the returned
+#' data/results. Method descriptions and expected outputs are described in
+#' `MsBackend.R`. This unit tests might be run for a newly developed backend
+#' extending `MsBackend` to ensure that it is compliant with it.
+#'
+#' To run this unit tests from another package:
+#'
+#' be <- <code to create and initialize the backend>
+#' test_suite <- system.file("test_backends", "test_MsBackend",
+#'     package = "Spectra")
+#' test_dir(test_suite, stop_on_failure = TRUE)
+#'
+#' The unit tests in this suite expect a variable `be` to be defined, which
+#' has to represent an **already initialized** backend instance.
+
+test_that("backend is valid", {
+    expect_true(validObject(be))
+})
+
+test_that("acquisitionNum", {
+    res <- acquisitionNum(be)
+    expect_true(is.integer(res))
+    expect_true(length(res) == length(be))
+})
+
+test_that("centroided", {
+    res <- centroided(be)
+    expect_true(length(res) == length(be))
+    expect_true(is.logical(res))
```

Comment:
use same order as above
```suggestion
    expect_type(res, "logical")
    expect_identical(length(res), length(be))
```